pypeit/core/gui/identify.py

```diff
@@ -269,7 +276,14 @@ def initialise(cls, arccen, lamps, slits, slit=0, par=None, wv_calib_all=None,
         fig, ax = plt.subplots(figsize=(16, 9), facecolor="white")
         plt.subplots_adjust(bottom=0.05, top=0.85, left=0.05, right=0.65)
         ax.add_line(spec)
-        ax.set_ylim((0.0, 1.1 * spec.get_ydata().max()))
+        if y_log:
+            ax.set_yscale('log')
+            ax.set_ylim( (max(1., spec.get_ydata().min()),
+                        4.0 * spec.get_ydata().max()))
+        else:
+            ax.set_ylim((0.0, 1.1 * spec.get_ydata().max()))
```

Comment:
Perhaps be explicit, and add:
`ax.set_yscale('linear')`

I can see reasons why we might want to toggle between log and linear during the fitting (e.g. the user... likely me... forgets to put the command line flag)

Reply:
Added.